components/ai_chat/core/browser/ai_chat_database.cc

```diff
@@ -1225,6 +1302,22 @@ bool AIChatDatabase::CreateSchema() {
     return false;
   }
 
+  static constexpr char kCreateUploadedImagesTableQuery[] =
+      "CREATE TABLE IF NOT EXISTS conversation_entry_uploaded_images("
+      "conversation_entry_uuid INTEGER NOT NULL,"
+      "image_order INTEGER NOT NULL,"
+      // encrypted filename
+      "filename BLOB NOT NULL,"
+      "filesize INTEGER NOT NULL,"
+      // encrypted image data
+      "image_data BLOB NOT NULL,"
+      "PRIMARY KEY(conversation_entry_uuid, image_order)"
+      ")";
+  CHECK(GetDB().IsSQLValid(kCreateUploadedImagesTableQuery));
```

Comment:
If we're planning on supporting any file type in https://github.com/brave/brave-browser/issues/44310 then should we make this table a little more flexible and have an enum file type field and a generic `data` field instead of `image_data` (and `order`)? That could save us having to migrate to a different table schema when implementing that feature. Seems we wouldn't want to use an additional table because of the `order` field.

Reply:
agree and fixed in 0a7e6d4a8a3468f69b55e5b552d7f3614232cff4